### PR TITLE
[Launch Your Store] Add e2e tests

### LINKED ISSUE #46568: Create e2e for coming soon and live modes
## Issue Description

Establish e2e tests for Launch Your Store.

## Acceptance Criteria

- Check live mode / coming soon settings have correct front-end rendering for non-logged in users.
- Check admin users see a banner.

### PR DESCRIPTION
**Pull request by psealock**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/46568

Add e2e tests around Launch Your Store. There are 4 scenarios tested to make sure the front end is displaying properly depending on Coming soon mode and logged in status.

* Entire site logged in and logged out
* Store only logged in and logged out

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Read through the added tests and code changes to make sure the logic is ok
2. See tests pass on CI

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [x] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

Add e2e test to unreleased feature

</details>

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/tests/shopper/launch-your-store.spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+				baseURL,
+				'woocommerce_coming_soon',
+				'no'
+			);
+		} catch ( error ) {
+			console.log( error );
+		}
+	} );
+
+	test.skip( 'Entire site coming soon mode', async ( { page, baseURL } ) => {
```
Comment (rjchow) [inline plugins/woocommerce/tests/e2e-pw/tests/shopper/launch-your-store.spec.js]: Tests are being skipped 😆

Comment (psealock) [reply plugins/woocommerce/tests/e2e-pw/tests/shopper/launch-your-store.spec.js]: Oops, fixed in https://github.com/woocommerce/woocommerce/pull/47239/commits/d9717391aacda778812b763d683dce81ee359082

**Review by rjchow — APPROVED** [review]
LGTM, thanks!

**Review by chihsuan — APPROVED** [review]
Nice work! LGTM. 💯

**Pull request merged**